docs/source/data_extraction.rst

```diff
@@ -177,27 +185,26 @@ For example:
 
     from time import sleep
     # 1. Submit the GEM Table build job
-    job = table_configs.build_ara_table(table_config)
+    job = project.tables.initiate_build(table_config)
     # 2. Poll the Job Status every second
     while True:
         status = project.table_configs.get_job_status(job.job_id)
```

Comment:
This code runs but issues warnings.  If correcting this is out of scope for this PR, we don't want it to go disappear off our radar.

Reply:
What's the warning message? I don't remember seeing one, but I often suppress warnings in my workflows.

Reply:
Just checked, I don't get one using `initiate_build` (using citrine v.0.61 though)

Reply:
Still a little confused on this - if it's a deprecated method, we should remove it from the docs. If not, we should make a card to address it. But I can't reproduce the warning you're getting.

Reply:
```
<ipython-input-11-6c84f4e184d2>:6: DeprecatedWarning: get_job_status is deprecated. Table build job status does not have a stable format. Please use TableCollection.build_from_config or TableCollection.get_by_build_job
  status = project.table_configs.get_job_status(job.job_id)
```
Issued under both Citrine 0.66 and 0.70.

Reply:
Oh so it's the part where we query for the job status. Yeah, if you know the correct way to do that that would be great!

Reply:
@kroenlein Did you look into this? I think the example should be something like:

```python
# 1. Submit the GEM Table build job
job = project.tables.initiate_build(table_config)
# 2. Get the table resource as an object
table = project.tables.get_by_build_job(job)
# 3. Download the table
project.tables.read(table, "./my_table.csv")
```

`get_by_build_job` already waits for the job to complete (and checks the status afterwards), so I don't think we don't need explicit polling.

Reply:
@latture I think it's smart to remove the polling, since the other method waits for completion. In this case, I think it's good to separate the `initiate_build` and `get_by_build_job` methods, since running them back to back will result in an error if you don't wait. 

See the a2d55e8 commit, Line 174-198 for my edits.

It still includes a reference to `get_job_status` in the text. Is there another way to tell if the build job has completed or not?

Reply:
> since running them back to back will result in an error if you don't wait.

Maybe I'm misunderstanding your comment, but why would you get an error if you run them back to back?

> It still includes a reference to get_job_status in the text. Is there another way to tell if the build job has completed or not?

I don't see a dedicated (non-deprecated) method to get the status. @kroenlein Do you know if one? Given a session and job ID you can get it using

```python
path = 'projects/{}/execution/job-status'.format(project_id)
params = {'job_id': job_id}
response = session.get_resource(path=path, params=params)
status: JobStatusResponse = JobStatusResponse.build(response)
```

but we generally try to avoid explicit session calls.

Reply:
I didn't get a chance to run with it.  I would argue we should probably have async demo code present as well, but the synchronous call example was also in the documentation.  The presumed path would be through the resources.job module in citrine-python.